### ISSUE TITLE
Relationship table accessibility

### ISSUE THREAD

**Issue by wintrymix** [dita-ot/dita-ot#2447]
When relationship tables are published to HTML, the links are formatted as a set of `div` elements. While this was acceptable under Web Content Accessibility Guidelines(WCAG) 1.0, WCAG 2.0 calls for lists of links to be formatted using list markup. HTML processing should be updated to conform to the recommendations of WCAG 2.0.

**Comment by robander** [comment]
I've gotten the same guidance from accessibility experts I work with, and we've had to override the default output to meet WCAG. I'm working on getting this one into the base code for 3.0 for HTML5 and XHTML.
